src/test/scala/rise/elevate/movement.scala

```diff
@@ -21,8 +21,10 @@ class movement extends test_util.Tests {
   def betaEtaEquals(a: Rise, b: Rise): Boolean = {
     val na = BENF(a).get
     val nb = BENF(b).get
-    val uab: Rise = toBeTyped(na) :: nb.t
-    makeClosed(uab) == makeClosed(nb)
+    val uab: Rise = eraseType(na) :: nb.t
```

Comment:
I am not sure why erasing types is necessary .. but otherwise this test randomly fails.

Reply:
Could you create an issue with a link to this so that we can investigate this?

Reply:
https://github.com/rise-lang/shine/issues/86